torch/quantization/fx/prepare.py

```diff
@@ -417,6 +431,44 @@ def maybe_insert_input_observers_for_node(
     node.args = tuple(new_args)
     node.kwargs = new_kwargs  # type: ignore[assignment]
 
+def maybe_insert_input_equalization_observers_for_node(
+    node: Node,
+    equalization_qconfig: Any,
+    model: torch.nn.Module,
+    modules: Dict[str, torch.nn.Module],
+    graph: Graph,
+    node_name_to_target_dtype: Dict[str, Any],
+) -> None:
+    """
+    If `node` needs to be equalized, find the input/weight observers it needs in
+    `equalization_qconfig`, creates them, and inserts it into `graph`.
+
+    If `node` does not need an equalization observer, returns None.
+    """
+    if equalization_qconfig is None or not node_supports_equalization(node, modules):
+        return
+
+    new_args = []
+    for arg in node.args:
+        if not isinstance(arg, Node) or node_arg_is_bias(node, arg):
+            continue
```

Comment:
for this case, should we assign `arg` to `new_args` instead of discarding?

Reply:
Yes! I will update that!